### ISSUE TITLE
andrpod-platform_tests timeout `gcloud firebase test android run --device model=redfin,version=30 --device model=starqlteue,version=26"`

### ISSUE THREAD

**Issue by jmagman** [flutter/flutter#125444]
```
Running command: "gcloud firebase test android run --type instrumentation --app build/app/outputs/apk/debug/app-debug.apk --test build/app/outputs/apk/androidTest/debug/app-debug-androidTest.apk --timeout 7m --results-bucket=gs://flutter_cirrus_testlab --results-dir=plugins_android_test/in_app_purchase/in_app_purchase/5665371445788672/3b34aa2f-7ed5-4c95-8319-26cd69b907f9/example/0/ --device model=redfin,version=30 --device model=starqlteue,version=26" in /tmp/cirrus-ci-build/packages/in_app_purchase/in_app_purchase/example
Timed out!
```
https://cirrus-ci.com/task/4665730331836416
https://cirrus-ci.com/task/5228680285257728

Looks just like https://github.com/flutter/flutter/issues/92607 (which is old and should probably be closed)

**Comment by keyonghan** [comment]
As there are not much useful info that can help us triage. I am following up with cirrus team to see if any more logs from their side.
(following up through email support as advised by cirrus team, this expects slow response)

**Comment by jmagman** [comment]
> As there are not much useful info that can help us triage. I am following up with cirrus team to see if any more logs from their side.

Can we talk to the firebase test team? Ideally we would have a contact to help with this...
cc @dnfield who always gets pinged because we don't have a real process solution to these timeouts (that I'm aware of)...

**Comment by dnfield** [comment]
I'm not sure how much help the firebase test team would be for this - the command appears to just be hanging. Maybe there's some way to increase verbosity on it temporarily?

In the past we've reached out to them when there's some other infra failure. We could try for this. I gave @keyonghan some contact information.

**Comment by stuartmorgan** [comment]
I just had a retry succeed, so maybe this was an outage that resolved on the FTL side?

**Comment by jmagman** [comment]
> Otherwise, I will try contacting firebase team.

It would be nice to have a POC for them since device outages seem to happen somewhat regularly.
Closing this particular incident.

**Comment by dnfield** [comment]
I don't think this was a device outage - usually those result in some kind of output and a test matrix we can look at and get an exit code reporting an infra failure.

This looks like `gcloud` just hung.
